src/sasjs-build/index.js

```diff
@@ -457,6 +474,190 @@ async function getDependencies(filePaths) {
   return dependenciesContent.join('\n')
 }
 
+export function getProgramList(fileContent) {
+  let fileHeader
+  try {
+    fileHeader = fileContent.split('/**')[1].split('**/')[0]
+  } catch (e) {
+    console.error(
+      chalk.redBright(
+        'File header parse error.\nPlease make sure your file header is in the correct format.'
+      )
+    )
+  }
+  let programsSection
+  try {
+    programsSection = fileHeader.split(/\<h4\> SAS Programs \<\/h4\>/i)[1]
+  } catch (e) {
+    console.error(
+      chalk.redBright(
+        'File header parse error.\nPlease make sure your SAS program dependencies are specified in the correct format.'
```

Comment:
it might be that the program does not have any SAS Programs to include - so the absence of this header doesn't mean anything, probably most programs won't have this

Reply:
We now return an empty list if the program doesn't have the comment header block.